docs/topics/i18n/timezones.txt

```diff
@@ -198,12 +202,15 @@ Create a view that can set the current timezone::
 
     from django.shortcuts import redirect, render
 
+    # Prepare a list of common timezone choices you wish to offer.
+    common_timezones = [...]
```

Comment:
include a concrete example - afaiu this should be a list of strings, not a list of `ZoneInfo`, which some might interpret the comment as implying